cypress/integration/e2e/table/undo-redo.spec.ts

```diff
@@ -0,0 +1,72 @@
+/// <reference path="../../../support/index.d.ts" />
+
+import { WorkerType } from "../../../../src/common-models/worker-info.model";
+import { GetWorkerShiftOptions } from "../../../support/commands";
+import { ShiftCode } from "../../../../src/common-models/shift-info.model";
+
+interface TestCase {
+  testedShift: GetWorkerShiftOptions;
+  firstShift: ShiftCode;
+  secondShift: ShiftCode;
+}
+
+const testCases: TestCase[] = [
+  {
+    testedShift: {
```

Comment:
Może lepiej nazwać to `testedCell/testedShiftCell`?

Reply:
Słuszna uwaga

Reply:
Podmieniłem

Reply:
ja też XD